src/ClockKit/CLKComplication.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Runtime.InteropServices;
+using Foundation;
+using CoreFoundation;
+using ObjCRuntime;
+
+namespace ClockKit {
+	public partial class CLKComplication {
+			[Watch (7,0)]
+			[DllImport (Constants.ClockKitLibrary)]
+			static extern IntPtr CLKAllComplicationFamilies ();
+
+			public static CLKComplicationFamily[] GetAllComplicationFamilies ()
+			{
+				using (var nsArray = new NSArray (CLKAllComplicationFamilies ())) {
+					var families = new CLKComplicationFamily [(int)nsArray.Count];
+					for (nuint i = 0; i < nsArray.Count; i++)
+					{
+						families[i] = (CLKComplicationFamily)nsArray.GetItem <NSNumber> (i).Int32Value;
+					}
+					return families;
+				}
+			}
```

Comment:
cute but 😛

```suggestion
			[Watch (7,0)]
			public static CLKComplicationFamily[] GetAllComplicationFamilies () => NSArray.EnumsFromHandle<CLKComplicationFamily> (CLKAllComplicationFamilies ());
```

Reply:
ah... source code beauty contest ;-)
I prefer the smallest (IL) one :)

Reply:
@spouliot heh I like consistency (I wonder who taught me that 😁) if we do (or don't) like `NSArray.EnumsFromHandle` then we have a problem elsewhere

Reply:
So, after my experience with LINQ... I'm going for the explicit implementation

Reply:
LINQ? There is no line involved at all :)

Reply:
@mandel-macaque if you mean `=>` that is just an [Expression body definition](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/statements-expressions-operators/expression-bodied-members) and if you really do not like it you can do 

```suggestion
			[Watch (7,0)]
			public static CLKComplicationFamily[] GetAllComplicationFamilies ()
			{
				return NSArray.EnumsFromHandle<CLKComplicationFamily> (CLKAllComplicationFamilies ());
			}
```

That said you are missing `[Watch (7,0)]` for `GetAllComplicationFamilies`

Reply:
No, I use => alot, I'm talking about the IL of EnumsFromHandle hehe

Reply:
a call to `EnumsFromHandle` is a lot smaller then the code above
the total size diff will depend on the app, and if anything else uses `EnumsFromHandle` or the code it depends on
so YYMV, but considering watchOS size limitations having less code is generally better